_docs/overview/overview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Forseti conceptual overview
+order: 001
+hide: 
+  left_sidebar: true
+---
+
+# {{ page.title }}
+
```

Comment:
Nit, I think we've standardized on no space between the heading and its content.

Reply:
I'm not sure what you mean. On other pages, I see # {{ page.title }} followed by a blank line before the content. Can you clarify?

Reply:
Ah, you're correct, I was wrong. Disregard.